assets/index.pug

```diff
@@ -0,0 +1,12 @@
+doctype html
```

Comment:
1/ this module is not meant to be an SPA, it's a collection of features that will be included in other SPA. There are 2 features:
- the videoconf setting in the calendar event edition form -> in esn-frontend-calendar
- the videoconf link in the inbox calendar bluebar -> in esn-frontend-inbox

Which means you should not have an index.pug (because it's not an SPA), and you should not have webpack configuration

Reply:
in `esn-frontend-common-libs`, I see that we still have `index.pug`. So we need to remove that file also, right?